skiko/src/iosMain/kotlin/org/jetbrains/skiko/SkikoUIView.kt

```diff
@@ -250,8 +250,11 @@ class SkikoUIView : UIView, UIKeyInputProtocol, UITextInputProtocol, UITextPaste
         )
     }
 
+    /**
+     * Attention! position may be null
+     */
     override fun positionFromPosition(position: UITextPosition, offset: NSInteger): UITextPosition? {
-        val p = (position as IntermediateTextPosition).position
+        val p = (position as? IntermediateTextPosition)?.position ?: 0
```

Comment:
shouldn't we return null if position isn't IntermediateTextPosition?

Reply:
Yeah, good idea!

Reply:
Done!